experimenter/experimenter/jetstream/models.py

```diff
@@ -201,6 +201,164 @@ def compute_significance(data_point: DataPoint) -> Significance:
 
 
 class ResultsObjectModelBase(BaseModel):
+    def __init__(
+        self,
+        result_metrics: dict[str, set[Statistic]],
+        data: JetstreamData,
+        experiment: NimbusExperiment,
+        window: AnalysisWindow = AnalysisWindow.OVERALL,
+    ):
+        super().__init__()
+
+        assert experiment.reference_branch
+
+        for jetstream_data_point in data:
+            metric = jetstream_data_point.metric
+            statistic = jetstream_data_point.statistic
+
+            if metric in result_metrics and statistic in result_metrics[metric]:
+                comparison_to_branch = jetstream_data_point.comparison_to_branch
+
+                branch_comparison = (
+                    BranchComparison.ABSOLUTE
+                    if jetstream_data_point.comparison is None
+                    else jetstream_data_point.comparison
+                )
+                data_point = DataPoint(
+                    lower=jetstream_data_point.lower,
+                    upper=jetstream_data_point.upper,
+                    point=jetstream_data_point.point,
+                )
+
+                # Set window_index (always 1 for OVERALL). Used by weekly DataPoint
+                # objects and for storing significance for each window.
```

Comment:
```suggestion
                # Used by weekly DataPoint
                # objects and for storing significance for each window.
```

The comment doesnt need to say "Setting X to Y", we can see that in the code.